modules/openvino_code/package.json

```diff
@@ -84,7 +83,7 @@
     "terser-webpack-plugin": "5.3.9",
     "ts-loader": "9.4.4",
     "typescript": "5.1.6",
-    "webpack": "5.88.2",
+    "webpack": "^5.96.1",
```

Comment:
Just double confirming that this ^ character is valid

Reply:
yes this ^ character is valid. It just shows that the webpack package has been updated automatically once you run npm install